content/blog/2024/2024-02-13-create-a-http-client-with-apache-http-client.md

```diff
@@ -0,0 +1,87 @@
+---
+authors: [sagaofsilence]
+title: "Create a HTTP Client with Apache HTTP Client"
+categories: ["Java"]
+date: 2024-02-11 00:00:00 +1100
+excerpt: "Get familiar with the Apache HTTP Client."
+image: images/stock/0063-interface-1200x628-branded.jpg
+url: create-a-http-client-with-apache-http-client
+---
+
+In this article series we're going to explore Apache `HTTP` client APIs. We'll get familiar with different ways Apache `HTTP` client enable developers to send and receive data over the internet. From simple `GET` requests to complex multipart `POST` requests, we'll cover it all with real-world examples.
```

Comment:
Let's write "Apache HttpClient" in all places where we talk about the Apache client as it's the official spelling on the website. In all places where we talk about HTTP clients in general, we can keep the current spelling.

Reply:
Done.